From: Helmut Grohne <helmut@subdivi.de>
Date: Mon, 24 Oct 2022 12:29:17 +0200
Subject: Use multiarch libdir

Change paths for libraries and pkg-config files to /usr/lib/<triplet>

Forwarded: not-needed
Bug-Debian: https://bugs.debian.org/#980335
---
 src/Makefile                | 4 ++--
 src/libsanlock.pc.in        | 4 ++--
 src/libsanlock_client.pc.in | 4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 533dd79..c7b8464 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -99,10 +99,10 @@ $(CMD_TARGET): $(LIBSO_ENTIRE_TARGET) $(CMD_SOURCE)
 	$(CC) $(CMD_CFLAGS) $(CMD_LDFLAGS) $(CMD_SOURCE) $(CMD_LDADD) -o $@ -L.
 
 $(LIBPC_ENTIRE_TARGET): $(LIBPC_ENTIRE_SOURCE)
-	sed -e "s/@VERSION@/$(VER)/" $(LIBPC_ENTIRE_SOURCE) > $(LIBPC_ENTIRE_TARGET)
+	sed -e "s/@VERSION@/$(VER)/" -e 's,@HEADIR@,$(HEADIR),' -e 's,@LIBDIR@,$(LIBDIR),' $(LIBPC_ENTIRE_SOURCE) > $(LIBPC_ENTIRE_TARGET)
 
 $(LIBPC_CLIENT_TARGET): $(LIBPC_CLIENT_SOURCE)
-	sed -e "s/@VERSION@/$(VER)/" $(LIBPC_CLIENT_SOURCE) > $(LIBPC_CLIENT_TARGET)
+	sed -e "s/@VERSION@/$(VER)/" -e 's,@HEADIR@,$(HEADIR),' -e 's,@LIBDIR@,$(LIBDIR),' $(LIBPC_CLIENT_SOURCE) > $(LIBPC_CLIENT_TARGET)
 
 clean:
 	rm -f *.o *.so *.so.* $(CMD_TARGET) $(LIBSO_ENTIRE_TARGET) $(LIBSO_CLIENT_TARGET) $(LIBPC_ENTIRE_TARGET) $(LIBPC_CLIENT_TARGET)
diff --git a/src/libsanlock.pc.in b/src/libsanlock.pc.in
index 4cbe0b5..32a600e 100644
--- a/src/libsanlock.pc.in
+++ b/src/libsanlock.pc.in
@@ -1,7 +1,7 @@
 prefix=/usr
 exec_prefix=${prefix}
-includedir=${prefix}/include
-libdir=${exec_prefix}/lib64
+includedir=@HEADIR@
+libdir=@LIBDIR@
 
 Name: libsanlock
 Description: The sanlock library
diff --git a/src/libsanlock_client.pc.in b/src/libsanlock_client.pc.in
index dcb877b..305f638 100644
--- a/src/libsanlock_client.pc.in
+++ b/src/libsanlock_client.pc.in
@@ -1,7 +1,7 @@
 prefix=/usr
 exec_prefix=${prefix}
-includedir=${prefix}/include
-libdir=${exec_prefix}/lib64
+includedir=@HEADIR@
+libdir=@LIBDIR@
 
 Name: libsanlock_client
 Description: The sanlock client library
